activation/post_verifier.go

```diff
@@ -25,11 +28,31 @@ type verifyPostJob struct {
 	result   chan error
 }
 
+type postStatesGetter interface {
+	Get() map[types.NodeID]types.PostState
+}
```

Comment:
NIT:

```suggestion
type postStates interface {
	States() map[types.NodeID]types.PostState
}
```

Reply:
It's `Get()` because the type implementing it has `Get()` and `Set()` methods.